engine/runtime/src/main/java/org/enso/interpreter/runtime/data/ArrayProxy.java

```diff
@@ -39,6 +41,17 @@ public ArrayProxy(long length, Object at) {
       }
     }
 
+    if (length < 0) {
+      CompilerDirectives.transferToInterpreter();
+      InteropLibrary interop = InteropLibrary.getUncached();
+      throw new PanicException(
+          Context.get(interop)
```

Comment:
Minor nit pick: This is equivalent to `Context.get(null)`, no need to get uncached interop lib just for that.